### PR TITLE
Update to almond 0.5.0

### PR DESCRIPTION
**Pull request by sbrunk**

### CODE REVIEW

File: binder/postBuild [hunk 1]
```diff
@@ -14,12 +14,13 @@ SCALA_VERSION=2.12.8 ALMOND_VERSION=0.4.0
   -o almond
 ./almond --install \
   --command "java -XX:MaxRAMPercentage=80.0 -jar almond" \
-  --copy-launcher true
+  --copy-launcher true \
+  --metabrowse true
```
Comment (alexarchambault) [inline binder/postBuild]: Just `--metabrowse` is enough (or `--metabrowse=true`, but `--metabrowse true` shouldn't work)

Comment (alexarchambault) [reply binder/postBuild]: Hmm, should be the same for `--copy-launcher` actually.

File: binder/postBuild [hunk 2]
```diff
@@ -28,7 +29,8 @@ SCALA_VERSION=2.11.12 ALMOND_VERSION=0.4.0
   -o almond-scala-2.11
 ./almond-scala-2.11 --install --id scala211 --display-name "Scala (2.11)" \
   --command "java -XX:MaxRAMPercentage=80.0 -jar almond-scala-2.11 --id scala211 --display-name 'Scala (2.11)'" \
-  --copy-launcher true
+  --copy-launcher true \
+  --metabrowse true
```
Comment (alexarchambault) [inline binder/postBuild]: Same as above.

File: binder/postBuild (hunk 1 above)
Comment (sbrunk) [reply binder/postBuild]: Ah thanks. Changed now.

**Review by alexarchambault — APPROVED** [review]

**Pull request merged**